languages/python/oso/polar/polar.py

```diff
@@ -214,9 +218,10 @@ def repl(self, files=[]):
 
             host = self.host.copy()
             host.set_accept_expression(True)
+            ffi_query.set_message_enricher(host.enrich_message)
             result = False
             try:
-                query = Query(ffi_query, host=host).run()
+                query = Query(ffi_query).run()
```

Comment:
Why did you remove this `host=`?

Reply:
Oh wow good catch -- scary that this doesn't cause any CI failures. I think I got my wires crossed when removing the `host` argument from the `ffi.Query` constructor. Will revert this

Reply:
Yeah unfortunately the `host` handling in here is very error prone. (Before your PR it was already a huge rough spot). If you have any thoughts about cleaning it up now that you've looked at it a bit it might be worth us spending time on.